showcase/app/templates/components/text.hbs

```diff
@@ -99,11 +107,12 @@
 
   <Shw::Text::H4 @tag="h3">Pre-defined colors</Shw::Text::H4>
   <Shw::Flex @direction="row" @wrap={{true}} {{style gap="2rem"}} as |SF|>
+    {{! @glint-expect-error }}
```

Comment:
For all these, could you just convert the route to `ts`?  I guess that's not the goal but it seems like an easy one to convert.

Reply:
We talked about this offline and have decided to move the changes in this file to a separate PR